### PR TITLE
refactor beats yaml config

### PR DESCRIPTION
**Pull request by zhongxp1219**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -28,7 +28,9 @@ go build .
 
 ### Usage
 
-#### Add following configuration to filebeat.yml
+In this section, you can use the sample config file in the directory [./sample/config/], or you can create it as follow steps.
+
+#### Add following configuration to filebeat.yml
```
Comment (hezhangjian) [inline README.md]: I think we can replace `filebeat.yml` to beat.yml. Since we already have metricbeat, heartbeat samples

**Comment by Shoothzj** [comment]
Since we already release other beats packages. I think we can remove `Build other beat` from usage.

**Review by hezhangjian — CHANGES_REQUESTED** [review]
Since we already release other beats packages. I think we can remove `Build other beat` from usage.

**Comment by zhongxp1219** [comment]
> Since we already release other beats packages. I think we can remove `Build other beat` from usage.

@Shoothzj  I think if we can reserve this section for others who want to `Build other beat` using source code ? Or we can remove usage section from README and let others follow our sample config - -

**Comment by hezhangjian** [comment]
@zhongxp1219 It should be other section, not the usage section. Like `build` section

File: README.md (hunk 1 above)
Comment (jiazhai) [reply README.md]: +1, filebeat.yml seems confusing. :)

**Comment by jiazhai** [comment]
Thanks, @zhongxp1219 for the great work. and thanks @Shoothzj for the review. 👍

**Comment by hezhangjian** [comment]
I think the original `filebeat.yml` can be deleted @zhongxp1219

**Review by hezhangjian — APPROVED** [review]

**Comment by hezhangjian** [comment]
Great work.

**Pull request merged**